src/test/java/systemtests/AddCommandSystemTest.java

```diff
@@ -48,14 +53,20 @@
     public void add() {
         Model model = getModel();
 
+        /* Login */
+        String loginCommand = LoginCommand.COMMAND_WORD + " ";
+        String command = loginCommand + PREFIX_USERNAME + VALID_MANAGER_USERNAME_ALICE
+                + " " + PREFIX_PASSWORD + VALID_MANAGER_PASSWORD_ALICE;
+        executeCommand(command);
+
```

Comment:
Could consider adding an assertCommandSuccess to make sure that the command is successful